autospec/avxjudge.py

```diff
@@ -0,0 +1,512 @@
+    avx2_instructions_hv.append("vpclmullqhqdq")
+
+    avx2_instructions_hv.append("vfmadd132ps")
+    avx2_instructions_hv.append("vfmadd213ps")
+    avx2_instructions_hv.append("vfmadd231ps")
+    avx2_instructions_hv.append("vfmadd132pd")
+    avx2_instructions_hv.append("vfmadd231pd")
+    avx2_instructions_hv.append("vfmadd213pd")
+    avx2_instructions_hv.append("vfmsub132ps")
+    avx2_instructions_hv.append("vfmsub213ps")
+    avx2_instructions_hv.append("vfmsub231ps")
+    avx2_instructions_hv.append("vfmsub132pd")
+    avx2_instructions_hv.append("vfmsub231pd")
+    avx2_instructions_hv.append("vfmsub213pd")
+
+    avx2_instructions_hv.append("vfnmadd132ps")
+    avx2_instructions_hv.append("vfnmadd213ps")
+    avx2_instructions_hv.append("vfnmadd231ps")
+    avx2_instructions_hv.append("vfnmadd132pd")
+    avx2_instructions_hv.append("vfnmadd231pd")
+    avx2_instructions_hv.append("vfnmadd213pd")
+    avx2_instructions_hv.append("vfnmsub132ps")
+    avx2_instructions_hv.append("vfnmsub213ps")
+    avx2_instructions_hv.append("vfnmsub231ps")
+    avx2_instructions_hv.append("vfnmsub132pd")
+    avx2_instructions_hv.append("vfnmsub231pd")
+    avx2_instructions_hv.append("vfnmsub213pd")
+    avx2_instructions_hv.append("vdivpd")
+
+
+    return
+
+def is_sse(instruction:str, args:str) -> float:
+
+    val: float = -1.0
+    if "xmm" in args:
+        if ("pd" in instruction or "ps" in instruction or instruction in sse_instructions_xmm):
+            val = 1.0
+        else:
+            val = 0.01
+    return val
+
+
+def is_avx2(instruction:str, args:str) -> float:
+    val: float = -1.0
+
+    if "ymm" in args:
+        if ("pd" in instruction or "ps" in instruction or instruction in avx2_instructions_ymm) and "xor" not in instruction and "vmov" not in instruction:
+            val = 1.0
+        else:
+            val = 0.01
+
+    if instruction in avx2_instructions_lv:
+        val = max(val, 0.1)
+    if instruction in avx2_instructions:
+        val = max(val, 1.0)
+    if instruction in avx2_instructions_hv:
+        val = max(val, 2.0)
+
+    return val
+
+def has_high_register(args: str) -> int:
+    if "mm16" in args:
+        return 1
+    if "mm17" in args:
+        return 1
+    if "mm18" in args:
+        return 1
+    if "mm19" in args:
+        return 1
+    if "mm20" in args:
+        return 1
+    if "mm21" in args:
+        return 1
+    if "mm22" in args:
+        return 1
+    if "mm23" in args:
+        return 1
+    if "mm24" in args:
+        return 1
+    if "mm25" in args:
+        return 1
+    if "mm26" in args:
+        return 1
+    if "mm27" in args:
+        return 1
+    if "mm28" in args:
+        return 1
+    if "mm29" in args:
+        return 1
+    if "mm30" in args:
+        return 1
+    if "mm31" in args:
+        return 1
+    return 0
+
+def is_avx512(instruction:str, args:str) -> float:
+    val: float = -1.0
+
+    if instruction in avx512_instructions_lv:
```

Comment:
`avx512_instruction_lv` is initialized to an empty list but is not populated anywhere, so this condition looks like it will always be False...

Is this an oversight, or is it reserved for future expansion of the code?